manage

```diff
@@ -692,11 +692,44 @@ themes.oscar() {
 }
 
 themes.simple() {
-    build_msg GRUNT "theme: simple"
-    npm --prefix searx/static/themes/simple run build
+    local src="searx/static/themes/simple/src"
+    local static="searx/static/themes/simple"
+    (   set -e
+        convert_if_newer "$src/svg/searxng-wordmark.svg" "$static/img/favicon.png" -background none -resize 64x64
```

Comment:
> :-1: The favicon.png does not have a alpha channel... ![image](https://user-images.githubusercontent.com/38733246/140829675-c9c53e1f-5ebc-4fc8-bdf9-3d8fd02ebd27.png) Is this intended? (Or to rephrase: is it possible to generate favicon.png with an alpha channel from the svg?)

Oops .. it is fixed right here by adding [`-background none`](https://www.imagemagick.org/script/command-line-options.php#background) to the convert command.

Reply:
I wish to keep all the build steps inside gruntfiles.js

In the simple theme directory, `npm run watch` allows to rebuild the theme a files change. If we mix bash and node it starts to be complicated to do that.

There are some grunt modules to convert to svg to png, but I haven't tried them.

Otherwise it is possible to call a command line from grunt: https://www.npmjs.com/package/grunt-exec

But, the logo won't change as often the other files, so IMO, it is not an issue if there is automatic build of the favicon.

Reply:
> I wish to keep all the build steps inside gruntfiles.js

Hm, .. I understand to want to have all theme build-tasks at one place (grunt), but the drawback I see is; with [grunt-exec](https://www.npmjs.com/package/grunt-exec) we can't express a conditional _convert if origin is newer_. IMO calling shell tasks from grunt increase complexity unneeded.